PointCloudProcessing/CMakeLists.txt

```diff
@@ -120,6 +126,7 @@ set(DEPENDENT_INCLUDE_DIRS
 	#${GeoTIFF_INCLUDE_DIR}
 	#${dep_libgeo_SOURCE_DIR}/include
 	#${dep_liblas_SOURCE_DIR}/include
+	${LAS_INCLUDE_DIR}/include

```

Comment:
Are you sure about this one?
Shouldn't it be {libLAS_INCLUDE_DIR}?
After the cmake configuration process you should be able to see all defined variables 
Check if LAS_INCLUDE_DIR existis

Reply:
Resolved: [f25f5ee](https://github.com/tumcms/Open-Infra-Platform/pull/523/commits/f25f5ee654502282c595af54562fcfc50caaffbd)